rendering_styles/default.render.xml

```diff
@@ -6802,6 +6807,7 @@
 						<case tag="tourism" value="lean_to" icon="lean_to"/>
 						<case tag="tourism" value="wilderness_hut" icon="wilderness_hut"/>
 						<apply shield="blue_round_shield"/>
+						<apply_if minzoom="14" moreDetailed="true" iconOrder="43" iconVisibleSize="9" intersectionSizeFactor="0.7"/>
```

Comment:
What was the purpose of this change? Why are you lowering iconOrder starting from zoom 14? Objects with iconOrder="43" will be rendered over iconOrder="81". Did you mean maxzoom="14" ?

Reply:
This is indeed wanted. I am lowering iconOrder at z14+ (and not z14- !) to ensure that mountain huts will have better priority to be rendered over other features starting from zoom 14, in case moreDetailed is also checked in Osmand settings.